jujugui/static/gui/src/test/test_app.js

```diff
@@ -120,6 +120,10 @@ function injectData(app, data) {
 
     it('should propagate login credentials from the configuration',
         function(done) {
+          window.juju_config = {
```

Comment:
instead of modifying the global state here, if you update how you're fetching this above you can pass these values into the app instantiation and they will be set as attributes.

Basically modifying the global state the tests execute in is always a bad idea.